### PR TITLE
Upgrade AGP to 8.0.2

### PR DESCRIPTION
**Pull request by Uldiniad**

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.1.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.0-all.zip
```
Comment (Uldiniad) [inline gradle/wrapper/gradle-wrapper.properties:6]: Keep in line with AGP recommended version

File: fake-store/build.gradle [hunk 2]
```diff
@@ -40,8 +40,8 @@ android {
     }
 
     compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_8
-        targetCompatibility JavaVersion.VERSION_1_8
+        sourceCompatibility JavaVersion.VERSION_17
+        targetCompatibility JavaVersion.VERSION_17
```
Comment (Uldiniad) [inline fake-store/build.gradle:44]: https://developer.android.com/build/releases/gradle-plugin#jdk-17-agp

Comment (ale5000-git) [reply fake-store/build.gradle:44]: I may be wrong but isn't just say that JDK 17 is now required to run Gradle?
You can run Gradle under the installed JDK 17 and compile for older version of Java or I'm wrong?

Comment (Uldiniad) [reply fake-store/build.gradle:44]: I'm not completely sure but I have noticed warnings (and in other projects errors) when using different java versions for different purposes

Comment (ale5000-git) [reply fake-store/build.gradle:44]: I cannot try currently but are you sure that with this change it will still work under Android 2.3?

Comment (Uldiniad) [reply fake-store/build.gradle:44]: I am not sure of that

**Comment by mar-v-in** [comment]
I close this pull request as the FakeStore source code is now maintained in the vending-app folder in GmsCore repository.

**Pull request closed without merging**